liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -418,7 +318,28 @@ public void recompileBuildFile(File buildFile, List<String> artifactPaths) {
                 log.debug("Could not recompile pom.xml", e);
             }
         }
+        
+        public int getMessageOccurrences(String regexp, File logFile) {
```

Comment:
As a best practice, add the `@Override` annotation since it is implementing an abstract method.